test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetListPackageTests.cs

```diff
@@ -721,7 +723,7 @@ public void DotnetListPackage_VerbositySwitchTogglesHttpVisibility(string args,
         [PlatformTheory(Platform.Windows)]
         [InlineData("true", false)]
         [InlineData("false", true)]
-        public async Task ListPackage_WithHttpSourceAndAllowInsecureConnections_WarnsCorrectly(string allowInsecureConnections, bool isHttpWarningExpected)
+        public async Task ListPackage_WithHttpSourceAndAllowInsecureConnections_LogsAnError(string allowInsecureConnections, bool isHttpErrorExpected)
```

Comment:
nit: I realize you're just updating these tests and they were already designed this way, but could you completely rewrite this test? 😄 If it's non-trivial let me know, but in the meantime, here's my ideas:

- The name of the test is repeating the test class name
- I don't see any reason to condition on `isHttpErrorExpected`. 
- Finally, it's not using FluentAssertions.

So, what about splitting this into 2 tests and removing the condition, as examples:
`Run_WithHttpSourceAndAllowInsecureConnections_Succeeds`
`Run_WithHttpSourceWithoutAllowInsecureConnections_LogsAnError`